fastly.yaml

```diff
@@ -0,0 +1,4 @@
+---
+vhost:
```

Comment:
Fair enough. How about `service`? A service can belong to an environment?

On 9 July 2015 at 16:12, Sam J Sharpe <notifications@github.com> wrote:

> In fastly.yaml
> <https://github.com/alphagov/fastly-configure/pull/1#discussion_r34265811>
> :
>
> > @@ -0,0 +1,4 @@
> > +---
> > +vhost:
>
> I know you've inherited vhost from the existing script, but since this is
> PR#1 that could quickly be changed to something else before it's in use by
> anyone else... ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/fastly-configure/pull/1/files#r34265811>.
>



-- 
Andrew Langhorn
Web Operations
Government Digital Service

e: andrew.langhorn@digital.cabinet-office.gov.uk
t: +44 (0)7810 737375
a: 6th Floor, Aviation House, 125 Kingsway, London, WC2B 6NH